advocacy_docs/edb-postgres-ai/databases/database_operations/resource_monitoring/memory_monitoring.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Basic memory monitoring 
+navTitle: Memory monitoring
+description: Perform basic memory monitoring.
+---
+
+A healthy memory configuration is essential for caching, buffering and maintenance operations. Therefore, it is important to ensure your memory configuration can cover the requirements of your applications, and can handle workload peeks.
+
+## Healthy memory configuration 
+
+As a general rule, a database server is considered to have a healthy memory configuration when the average **Memory** usage is stable and does not surpass the 80% threshold of the allotted memory resources over a longer period of time. This ensures the database server isn't overburdened.
+
+The key determining factor is the stability of caching, buffering and maintenance operations. If you expect an exponential increase of these operations, 80% could soon not be enough.
+
+If you don't expect an exponential increase of caching, buffering and maintenance operations, 90% of used disk capacity is acceptable.
+
+## Symptoms of memory issues 
+
+Some symptoms that you could encounter when the memory is not sufficient are:
+
+- Slow response times 
+- Frequen checkpoints, which consume more resources
+- High CPU usage
+- Connection issues
+- Other
```

Comment:
I think these are a little muddled. AFAIU the only symptom of high memory consumption is that there is less memory available for system file cache, so I/O speed might drop off. The main issue is that when you actually run out of memory Postgres crashes. So the main objective of monitoring memory is to make sure you have enough headroom to avoid that happening.

We should also talk here about how memory is allocated to Postgres in more specific details - i.e. a fixed shared_buffer, some multiple of maintaince_work_mem, plus some less predictable multiple of active_sessions*work_mem.

Reply:
What about getting rid of this section in all pages, and replacing it with a short explanation of what could happen if that specific metric is at 100%. For example, full memory means the Postgres database crashes. Full Disk IOPS would probably mean high latency and long wait times, etc. WDYT?